be/src/exprs/aggregate_functions.cpp

```diff
@@ -654,22 +654,6 @@ void AggregateFunctions::min(FunctionContext*, const DecimalV2Val& src, DecimalV
     }
 }
 
-template <>
-void AggregateFunctions::min(FunctionContext*, const LargeIntVal& src, LargeIntVal* dst) {
-    if (src.is_null) {
```

Comment:
why delete this function? LargeIntValue is not used?

Reply:
The implementation for LargeIntValue is totally duplicate with the template function of min/max, I removed these code and add a template instance for LargeIntValue type, see L2543 and L2593